src/background.js

```diff
@@ -824,19 +824,7 @@ function onMessageHandler(request, sender, callback) {
 	if (name === 'account.getUserSubscriptionData') {
 		account.getUserSubscriptionData()
 			.then((customer) => {
-				// TODO temporary fix to handle multiple subscriptions
-				let sub = customer.subscriptions;
-				if (!Array.isArray(sub)) {
-					sub = [sub];
-				}
-				const subscriptionData = sub.reduce((acc, curr) => {
-					let a = acc;
-					if (curr.productName.includes('Plus')) {
-						a = curr;
-					}
-					return a;
-				}, {});
-				callback({ subscriptionData });
+				callback({ subscriptionData: customer });
```

Comment:
`Account#getUserSubscriptionData` now returns a subscription object if the user is plus/premium, or the customer object if user is neither. So it looks like we have changed what we pass to the callback if the user is not a subscriber. Is this as intended?

Reply:
Ah. I was trying to reduce code duplication between account#getUserSubscriptionData and the background, but it looks like I got a little too fancy since that function is used in several places in the background class. Changed function back to it's original behavior

Reply:
See my further comments below :)